themes/docs-new/layouts/shortcodes/unified_mode.md

```diff
@@ -0,0 +1,192 @@
+<!-- markdownlint-disable MD041 -->
+Unified mode combines the compile and converge stages of the Chef Infra Client run into one phase. In unified mode, Chef Infra Client compiles and converges a custom resource in one pass and in the order that the code is compiled, from the beginning to the end.
+
+Unified mode is only available on custom resources. Chef Infra Client processes all other resources with separate compile and converge phases.
+
+In Chef Infra Client 17 (April 2021) and some earlier versions, the default behavior is `unified_mode false`. Enable unified mode on a custom resource with `unified_mode true`. Chef Infra Client displays a deprecation message with `unified_mode false`.
+
+In Chef Infra Client 18 (April 2022), `unified_mode true` will become the default behavior.
+
+|Chef Infra Client  | Unified Mode                 |
+|-------------------|-----------------------------|
+| 18.x (2022)       | Default: `unified_mode true` |
+| 17.x (2021)              | Default: `unified_mode false`|
+| 16.x (2020)              | Default: `unified_mode false`|
+| 15.3 and higher   | Default: `unified_mode false`|
+| 15.0–15.2        | Not available               |
+| 14.14 and higher  | Default: `unified_mode false`|
+| Lower than 14.14  | Not available               |
+
+## Enable Unified Mode
+
+Enable unified mode by adding the `unified_mode true` in the custom resource. You can upgrade most custom resources to use unified mode without additional work other than testing and validation.
+
+```ruby
+# enable unified mode
+unified_mode true
+
+provides :myresource
+
+actions :run do
+  [...]
+end
+
+```
+
+## Troubleshoot Unified Mode
+
+Unified mode changes the execution of a custom resource to run in one phase, in the order that the code is written, from first line of the code to the last. Custom resources designed to use two phases may need modification. These fall into three general types:
+
+- Resources with changes to internal sub-resources
+- Resources with actions on later resources
+- Resources that rely on the out-of-order execution
+
+When designing a custom resource for unified mode:
+
+- Declare a resource first and then declare actions on it
+- Write resources in run-time order
+
+### Resources with Changes to Internal Sub-resources
```

Comment:
```suggestion
### Resources With Changes to Internal Sub-Resources
```

Reply:
It looks strange, but that's correct for Chicago. In AP we'd capitalize `With` but not `resources`.
We should look into the Hugo title-case feature. I tried it once & couldn't get it to work.